data/human/news.txt

```diff
@@ -1557,6 +1557,28 @@ news "republic soldier"
 
 
 
+news "lawyer"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	name
+		word
+			"Lawyer"
+			"Attorney"
+	message
+		word
+			`"`
+		word
+			"The Dereliction and Salvage Act of 2976 states unmanned vessels gain abandoned status as soon as their owners leave the system."
+			"The Capable Rescue Act of 2318 requires all manned vessels to be capable of individual spaceflight in case of an emergency."
+			"In 2537, the Parliament voted to classify crewing spaceships as hazardous duty, raising the crew's minimum wage to a 100 credits. Their salary has not changed since."
```

Comment:
Since the Republic wasn't founded until the 2700s, Parliament might not have existed until then and couldn't have voted it in. But, it could be interesting to mention an earlier 'common interstellar law' or an interstellar convention, or the origins of a law (the motive or the area it came from).

Reply:
The second one was just a random number. The first one was actually based on the Alpha wars:
> 	"I see," says the officer. "Of course, what you deserve even more thanks for is your assistance during the Second Alpha War. Here's a photo of you from 2461 AD." He displays another photo, again apparently of Hanover himself, with some men in archaic pre-Republic uniforms. The guards standing behind you raise their weapons and point them at you and Elias.

There must have been some regulations before that, so that's why I used something from the previous century. I'm fine with changing that, though.
```suggestion
			"The Capable Rescue Act of 2818 requires all manned vessels to be capable of individual spaceflight in case of an emergency."
			"In 2937, the Parliament voted to classify crewing spaceships as hazardous duty, raising the crew's minimum wage to a 100 credits. Their salary has not changed since."
```

Reply:
So... should I just revert the first one?

Reply:
Err, if it could be changed to have some context about how it was a pre-Republic law.

Reply:
Something like this, maybe?

> "The Capable Rescue Act of 2318 requires all manned vessels to be capable of individual spaceflight in case of an emergency. This is the oldest law still recognized today, predating the Republic by four centuries."